package.json

```diff
@@ -3,27 +3,29 @@
   "version": "3.2.1",
   "description": "Predictable state container for JavaScript apps",
   "main": "lib/index.js",
-  "jsnext:main": "src/index.js",
+  "jsnext:main": "es/index.js",
   "files": [
     "dist",
     "lib",
+    "es",
     "src"
   ],
   "scripts": {
     "clean": "rimraf lib dist coverage",
     "lint": "eslint src test examples",
-    "test": "mocha --compilers js:babel-register --recursive",
+    "test": "cross-env BABEL_ENV=commonjs mocha --compilers js:babel-register --recursive",
     "test:watch": "npm test -- --watch",
-    "test:cov": "babel-node $(npm bin)/isparta cover $(npm bin)/_mocha -- --recursive",
-    "test:examples": "babel-node examples/testAll.js",
-    "check:lib": "npm run lint && npm run test",
+    "test:cov": "cross-env BABEL_ENV=commonjs babel-node $(npm bin)/isparta cover $(npm bin)/_mocha -- --recursive",
+    "test:examples": "cross-env BABEL_ENV=commonjs babel-node examples/testAll.js",
+    "check:src": "npm run lint && npm run test",
     "check:examples": "npm run build:examples && npm run test:examples",
-    "build:lib": "babel src --out-dir lib",
-    "build:umd": "cross-env NODE_ENV=development webpack src/index.js dist/redux.js",
-    "build:umd:min": "cross-env NODE_ENV=production webpack src/index.js dist/redux.min.js",
-    "build:examples": "babel-node examples/buildAll.js",
-    "build": "npm run build:lib && npm run build:umd && npm run build:umd:min && node ./prepublish",
-    "prepublish": "npm run clean && npm run check:lib && npm run build",
+    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
+    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
```

Comment:
Good stuff :) thanks for the clarification!
On Fri 5 Feb 2016 at 23:51 Dan Abramov <notifications@github.com> wrote:

> In package.json
> <https://github.com/rackt/redux/pull/1369#discussion_r52088759>:
>
> >      "check:examples": "npm run build:examples && npm run test:examples",
> > -    "build:lib": "babel src --out-dir lib",
> > -    "build:umd": "cross-env NODE_ENV=development webpack src/index.js dist/redux.js",
> > -    "build:umd:min": "cross-env NODE_ENV=production webpack src/index.js dist/redux.min.js",
> > -    "build:examples": "babel-node examples/buildAll.js",
> > -    "build": "npm run build:lib && npm run build:umd && npm run build:umd:min && node ./prepublish",
> > -    "prepublish": "npm run clean && npm run check:lib && npm run build",
> > +    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
> > +    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
>
> I just left it for consistency so I always specify either that, or
> commonjs. Indeed it is not currently used explicitly anywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rackt/redux/pull/1369/files#r52088759>.
>